components/announcement_bar/default_announcement_bar/announcement_bar.jsx

```diff
@@ -126,9 +142,14 @@ export default class AnnouncementBar extends React.PureComponent {
                     overlay={announcementTooltip}
                 >
                     <span>
+                        {this.props.showLinkAsButton &&
+                            (this.props.showCloseButton ? <div className={'content__icon'}>{''}</div> : <div className={'content__icon'}>{''}</div>)
```

Comment:
What's the character you're using here?

Reply:
It's what Figma spit out for the ! icon: . If you click the "Content" it copies to clipboard. I think github is having issues displaying unicode? 
![image](https://user-images.githubusercontent.com/12176405/94281372-8a23f180-ff1c-11ea-80d6-64427d31566a.png)

Reply:
Oh okay, I usually just use an `<i>` tag, but if that works that's fine.